package.json

```diff
@@ -66,7 +66,7 @@
     "babel-loader": "^8.2.2",
     "babel-plugin-transform-class-properties": "^6.24.1",
     "chalk-rainbow": "^1.0.0",
-    "chromatic": "^5.9.2",
+    "chromatic": "^5.10.0-canary.0",
```

Comment:
`canary` 😨 - I'm actually happy to move forward if it's working, I'm surprised npm offered it though

Reply:
woops, didn't notice that, i'll downgrade to 5.9.2 until 5.10.0 is released proper